app/pages/settings/email.jsx

```diff
@@ -0,0 +1,289 @@
+import React from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import talkClient from 'panoptes-client/lib/talk-client';
+import AutoSave from '../../components/auto-save';
+import handleInputChange from '../../lib/handle-input-change';
+
+class EmailSettingsPage extends React.Component {
+  constructor(props) {
+    super(props);
+    console.log(props)
+    this.state = {
+      page: 1,
+      projects: [],
+      projectPreferences: []
+    };
+    this.handlePreferenceChange = this.handlePreferenceChange.bind(this);
+    this.nameOfPreference = this.nameOfPreference.bind(this);
+    this.sortPreferences = this.sortPreferences.bind(this);
+    this.talkPreferenceOption = this.talkPreferenceOption.bind(this);
+    this.getProjectForPreferences(props.user);
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (!this.state.page === prevState.page) {
+      this.getProjectForPreferences();
+    }
+  }
+
+  getProjectForPreferences(user) {
+    return user.get('project_preferences', {
+      page: this.state.page,
+      sort: 'display_name'
+    })
+    .then((preferences) => {
+      if (preferences.length > 0) {
+        const projectIDs = preferences.map(pref => pref.links.project);
+        apiClient.type('projects').get({
+          id: projectIDs
+        })
+        .then((projects) => {
+          return this.setState({
+            meta: preferences[0].getMeta(),
+            projectPreferences: preferences,
+            projects
+          });
+        });
+      }
+    })
+    .catch((error) => {
+      console.log('Something went wrong. Error: ', error);
+    });
+  }
+
+  handleProjectEmailChange(projectPreference, args) {
+    handleInputChange.apply(projectPreference, args);
+    projectPreference.save();
+    this.setState();
+  }
+
+  handlePreferenceChange(preference, event) {
+    return preference.update({
+      email_digest: event.target.value
+    })
+    .save();
+  }
+
+  nameOfPreference(preference) {
+    switch (preference.category) {
```

Comment:
This whole block can probably be replaced by a dictionary object and a lookup.
```
{
  participating_discussions: "When discussions I'm participating in are updated.',
  followed_discussions: "When discussions I'm following are updated."
  etc.
}
```

Also, the English text strings should be moved into counterpart, but maybe not in this PR?

Reply:
@eatyourgreens I'm happy to move the English strings into a counterpart. They'd include zoo-wide preferences and talk preferences. Anything else?

Reply:
All the form labels, I guess. 

Could you add Italian translations too? I think you can test by adding `language=it` as a query parameter.

Reply:
I probably need to be reminded how to do that. I'll push something shortly and we discuss from there, thanks.

Reply:
Italian strings are in `locales/it.json`.